src/app/store/configureStore.js

```diff
@@ -0,0 +1,9 @@
+import { createStore } from 'redux';
+import rootReducer from './root-reducer';
+
+export default function configureStore(initialState) {
+    return createStore(
+        rootReducer,
+        initialState,
+        window.devToolsExtension ? window.devToolsExtension() : f => f);
```

Comment:
I think this initialization of dev tools extensions is outdated.
See https://github.com/zalmoxisus/redux-devtools-extension#13-use-redux-devtools-extension-package-from-npm (`composeWithDevTools `).